packages/backend/src/modules/tvl2/modules/Tvl2Module.ts

```diff
@@ -90,11 +92,26 @@ export function createTvl2Module(
   const statusRouter = createTvl2StatusRouter(config.tvl2, clock)
   const tvlRouter = createTvl2Router(tvlController, clock)
 
+  const tvlCleaner = new TvlCleaner(
+    clock,
+    logger,
+    peripherals.getRepository(TvlCleanerRepository),
+    [
+      peripherals.getRepository(AmountRepository),
```

Comment:
clean also BlockTimestamps - for the future: to see all the tables used by tvl2 navigate to repositories folder